.changeset/good-years-bake.md

```diff
@@ -0,0 +1,5 @@
+---
+'@finos/legend-studio': patch
```

Comment:
Also, I think we can safely put this one as a `minor` version bump. This is not a trivial patch right?

Reply:
yeah agree this should be a minor increase